Intersect.Client/Interface/Game/EventWindow.cs

```diff
@@ -123,6 +143,8 @@ public void Update()
                         maxResponse = 4;
                     }
 
+                    Typewriting = ClientConfiguration.Instance.EnableTypewriting && Globals.Database.TypewriterText;
```

Comment:
What happens if it renders a typewriting event window, you go change the settings to turn it off, and then you trigger another event window? And vice-versa?

This is more a question of "Does EventWindow get disposed and recreated, or reused?"

Reply:
This runs everytime a _new_ dialog is displayed. So you will see the changes on the _next_ dialog. We always hide dialogs before showing the next one, so this code gets hit each time a new one appears.